packages/components/Callout/src/Callout.types.ts

```diff
@@ -35,7 +35,14 @@ export interface RestoreFocusEvent {
   };
 }
 
-export interface ICalloutTokens extends IBackgroundColorTokens, IBorderTokens {
+export interface OmittedBorderTokens {
```

Comment:
nit, you don't need to export this. CalloutBorderTokens probably doesn't need to be exported either?